tfhe/src/shortint/ciphertext/mod.rs

```diff
@@ -131,6 +131,9 @@ pub struct CiphertextBase<OpOrder: PBSOrderMarker> {
 pub type CiphertextBig = CiphertextBase<KeyswitchBootstrap>;
 pub type CiphertextSmall = CiphertextBase<BootstrapKeyswitch>;
 
+#[derive(Debug)]
+pub struct CiphertextExplicitConversionError {}
```

Comment:
let's carry the input ciphertext op order and the output op order and implement format with a nice error message please !

Reply:
or even use generics here if you want to and make sure everything gets inlined/monomorphized at compile time

Reply:
How about now ?

Reply:
Looks good thanks !